papaparse.js

```diff
@@ -1100,7 +1103,10 @@ License: MIT
 		{
 			_paused = true;
 			_parser.abort();
-			_input = _input.substring(_parser.getCharIndex());
+
+			// If it is streaming via "chunking", the reader will start appending correctly already so no need to substring,
+			// otherwise we can get duplicate content within a row
+			_input = isFunction(_config.chunk) ? "" : _input.substring(_parser.getCharIndex());
```

Comment:
My original recommendation in the bug ticket was to just set this to an empty string or remove the usage of `_input` all together. Turns out `step` needs this while `chunk` doesn't.